django-stubs/contrib/postgres/fields/array.pyi

```diff
@@ -17,17 +17,18 @@ class ArrayField(CheckFieldDefaultMixin, Generic[_V], Field[_V | Combinable, _V]
     from_db_value: Any = ...
     base_field: Field[_V, _V] = ...
     @overload
-    def __new__(  # type: ignore [misc]
+    def __new__(
         cls,
         base_field: Field[Any, _V],
         size: int | None = ...,
         verbose_name: str | None = ...,
+        *,
         name: str | None = ...,
         primary_key: bool = ...,
         max_length: int | None = ...,
         unique: bool = ...,
         blank: bool = ...,
-        null: Literal[False] = ...,
+        null: Literal[False] = False,
```

Comment:
I think we still want these to be `...`, I think in general type stubs don't specify the actual default value

Reply:
if i understood it correctly, then we can't do `...` more. We definitely need a default value, for the correct choice of `@overload`. https://github.com/microsoft/pyright/issues/8860 was closed as "as designed", so i think stubs should accept this.

Reply:
> I think we still want these to be `...`, I think in general type stubs don't specify the actual default value

@sbdchd 
How strong is your commitment to `...`? Did the new rules in `pyright` introduce a bit of doubt?

The dev version of `pylance` already includes the new `pyright`, in a month it will appear for all users.

After which nullable fields will "break" in their projects, let's fix them in advance.